.github/workflows/release.yml

```diff
@@ -6,7 +6,9 @@ on:
       - main
   pull_request:
 
-concurrency: ${{ github.workflow }}-${{ github.ref }}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Not sure how safe it is to cancel a running release workflow. 🤔 

But I guess, even if the release did not complete altogether - the release CI that superseeds should.

Reply:
Reverted 👍